DashAI/front/src/components/DatasetsTable.jsx

```diff
@@ -0,0 +1,205 @@
+import React, { useEffect } from "react";
+import PropTypes from "prop-types";
+
+import { DataGrid, GridActionsCellItem } from "@mui/x-data-grid";
+import DeleteIcon from "@mui/icons-material/Delete";
+import EditIcon from "@mui/icons-material/Edit";
+import AddIcon from "@mui/icons-material/Add";
+import {
+  Button,
+  Grid,
+  Paper,
+  Typography,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from "@mui/material";
+
+function EditDataset() {
+  // const [open, setOpen] = React.useState(false);
+  const handleEdit = () => {};
+  return (
+    <GridActionsCellItem
+      key="edit-button"
+      icon={<EditIcon />}
+      label="Edit"
+      onClick={handleEdit}
+      sx={{ color: "#f1ae61" }}
+    />
+  );
+}
+
+function DeleteDataset({ deleteFromTable }) {
+  const [open, setOpen] = React.useState(false);
+  const handleDelete = () => {
+    deleteFromTable();
+    setOpen(false);
+  };
+  return (
+    <React.Fragment>
+      <GridActionsCellItem
+        key="delete-button"
+        icon={<DeleteIcon />}
+        label="Delete"
+        onClick={() => setOpen(true)}
+        sx={{ color: "#ff8383" }}
+      />
+      <Dialog
+        open={open}
+        onClose={() => setOpen(false)}
+        aria-labelledby="alert-dialog-title"
+        aria-describedby="alert-dialog-description"
+      >
+        <div style={{ backgroundColor: "#2e3037" }}>
```

Comment:
Is it necessary to specify the color of the text here?
Is it not inferred from the main theme?

Why don't we use the default ones? It is much easier to maintain color consistency this way.